examples/workflow/1_create_model.ipynb

```diff
@@ -0,0 +1,349 @@
+{
+ "cells": [
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    "Download this notebook :download:`here <1_create_model.ipynb>`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Step 1: Create an amplitude model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Your analysis starts by defining an amplitude module that describe the intermediate resonances in the decay process you study. For this, of course, you first have to determine what resonances there could be.\n",
+    "\n",
+    "ComPWA comes with a so-called ['expert system'](https://en.wikipedia.org/wiki/Expert_system) that can determine which resonances are allowed for a problem set you define (initial state, final state, allowed interactions, etc.). The system investigates all decay chain combinations and checks whether all quantum numbers comply with conservation rules.\n",
+    "\n",
+    "Next, the expert system writes out an amplitude model that represents the interference patterns of allowed resonances. The model can for instance be used to generate a data set (toy Monte-Carlo) for this decay and you optimize its parameters you perform the fit to actual data."
```

Comment:
This section has to be rewritten a bit to be more precise. I will post the snippet here later

Reply:
Your analysis starts by defining an amplitude model, that describe the reaction process you want to study. Such a model generally is very complex and requires a fair amount of effort by the analyist (you). This also gives a lot of rooms for mistakes.  

To facilitate this process of defining an amplitude model, pycompwa comes with a so-called ‘expert system’. Its responsibility is to give you advice in form of an amplitude model based on the problem set you define (initial state, final state, allowed interactions, intermediate states, etc.). Internally, the system propagates the quantum numbers through the reaction graph while satisfying the specified conservation rules. How to control this procedure is explained below in more detail.

Afterwards, the amplitude model of the the expert system can be exported. The model can for instance be used to generate a data set (toy Monte-Carlo) for this reaction and to optimize its parameters to resemble an actual data set as good as possible.

Reply:
Feel free to correct spelling mistakes or restructure if you think some stuff is unclear

Reply:
https://github.com/ComPWA/pycompwa/pull/74/commits/80c86eade68356298dd95fbc8f416e5140661a8a